lib/quick_travel/cache.rb

```diff
@@ -10,6 +10,8 @@ def self.cache(key, cache_options = {})
       cached_value = cache_store.read(key)
       return cached_value unless cached_value.nil?
       return nil unless block_given?
+      cache_options ||= {}
```

Comment:
Yes it's needed, without the specs fail. This is because the caller (QuickTravel::Adapter.all) does this, where it's usually nil (no cache options)

Are there any ways to fix the caller?

```ruby
        QuickTravel::Cache.cache(opts[:cache], opts[:cache_options]) {
          get_and_validate(request_path, opts.except(:cache, :cache_options))
        }
```